tfhe/src/core_crypto/fft_impl/crypto/bootstrap.rs

```diff
@@ -320,3 +321,114 @@ pub fn pbs_modulus_switch<Scalar: UnsignedTorus + CastInto<usize>>(
     output <<= lut_count_log.0;
     <Scalar as CastInto<usize>>::cast_into(output)
 }
+
+pub trait FourierBootstrapKey<Scalar> {
```

Comment:
can maybe put in some shared module ? Not a blocker

Reply:
yeah, though i don't know where i should put it

Reply:
may I suggest having something like

fft_impl
|---- f64 (then crypto, etc)
|---- f128
|---- common ?

or

fft_impl
|---- crypto
|       |---- f64
|       |---- f128
|---- common

so either one directory with the crypto/math etc folder for each kind of bootstrap and a common dir in the fft_impl

or a single crypto/math tree with both f64 and f128 variants every time

Not sure which one is better